superset-frontend/package.json

```diff
@@ -114,7 +113,12 @@
     "@superset-ui/plugin-chart-word-cloud": "file:./plugins/plugin-chart-word-cloud",
     "@superset-ui/preset-chart-xy": "file:./plugins/preset-chart-xy",
     "@superset-ui/switchboard": "file:./packages/superset-ui-switchboard",
+    "@visx/axis": "^3.0.1",
+    "@visx/grid": "^3.0.1",
```

Comment:
any reason for/against using the same version across all @vizx packages?

Reply:
oh good catch. will update all visx to 3.0.1

Reply:
tried to install @visx/xychart 3.0.1 but there was a react-spring error